docs/sysadmin-basics/index.txt

```diff
@@ -3,7 +3,7 @@
 ########
 
 Two main components are required to use Determined: the :ref:`Determined CLI <install-cli>` and a
-:ref:`Determined cluster <install-cluster>`. All users should install the Determined CLI on their
+:ref:`Determined cluster <install-cluster>`. Install the Determined CLI on your
```

Comment:
What is improved about this wording vs the last wording?

Reply:
Changed to meet style guide requirements. Makes a stronger sentence and shouldn't have multiple 'you' in a sentence; also don't reference 'user'. Better to speak to reader, directly.